clang/lib/CConv/ConstraintVariables.cpp

```diff
@@ -250,6 +250,20 @@ PointerVariableConstraint::PointerVariableConstraint(const QualType &QT,
       ArrPresent = IsArr = true;
       IsIncompleteArr = Ty->isIncompleteArrayType();
 
+      // Boil off the typedefs in the array case.
```

Comment:
I don't understand the big picture, but: Is removing the typedef a problem, i.e., in conflict with the goals of the other work to preserve typedefs during rewriting?

Reply:
Yes, this will have to change to get typedefs to work. We would not want to boil off the typedefs (though I think we would still want to boil off ParenTypes)

Reply:
OK, I guess this is fine for now, then.